src/Microsoft.DotNet.Build.Tasks/PackageFiles/Build.Common.props

```diff
@@ -37,6 +37,7 @@
 
   <!-- Common BuildTools properties -->
   <PropertyGroup>
+    <RootIntermediateOutputPath>$(MSBuildThisFileDirectory)../bin/obj/</RootIntermediateOutputPath>
```

Comment:
You should have a condition here, so someone can default it first:

`Condition="'$(RootIntermediateOutputPath)'==''`

This is typically good practice in MSBuild.  If someone else set this property, don't overwrite what they set.